scripts/tests/localsetup_tests.py

```diff
@@ -403,21 +403,21 @@ def test_opbeans_loadgen(self):
             no_opbeans_node_loadgen=True,
             opbeans_python_loadgen_rpm=50,
             opbeans_ruby_loadgen_rpm=10,
-        ).render()
-        assert opbeans_load_gen == yaml.load("""
-            opbeans-load-generator:
-                image: opbeans/opbeans-loadgen:latest
-                container_name: localtesting_6.3.1_opbeans-load-generator
-                depends_on:
-                    opbeans-python: {condition: service_healthy}
-                    opbeans-ruby: {condition: service_healthy}
-                environment:
-                 - 'OPBEANS_URLS=opbeans-python:http://opbeans-python:3000,opbeans-ruby:http://opbeans-ruby:3000'
-                 - 'OPBEANS_RPMS=opbeans-python:50,opbeans-ruby:10'
-                logging:
-                    driver: json-file
-                    options: {max-file: '5', max-size: 2m}""")
+        ).render()["opbeans-load-generator"]
+
+        self.assertEqual("opbeans/opbeans-loadgen:latest", opbeans_load_gen["image"])
```

Comment:
you can just use 

    assert "opbeans/opbeans-loadgen:latest" == opbeans_load_gen["image"]

and similar for the other asserts

Reply:
Awesome, thanks, I'll update it